pkgs/by-name/en/encore/go.nix

```
@@ -0,0 +1,87 @@
+{ fetchFromGitHub
+, stdenv
+, lib
+, go
+, ...
+}:
+stdenv.mkDerivation rec {
```

Use of `finalAttrs` is preferred over `rec` where possible. Remember to close the newly opened parenthesis at the end of the file:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```